apps/speed.c

```diff
@@ -1848,9 +1895,9 @@ int speed_main(int argc, char **argv)
     e = setup_engine(engine_id, 0);
 
     /* No parameters; turn on everything. */
-    if (argc == 0 && !doit[D_EVP] && !doit[D_EVP_HMAC]) {
+    if (argc == 0 && !doit[D_EVP] && !doit[D_EVP_HMAC] && !doit[D_EVP_CMAC]) {
         for (i = 0; i < ALGOR_NUM; i++)
-            if (i != D_EVP && i != D_EVP_HMAC)
+            if (i != D_EVP && i != D_EVP_HMAC && 1 != D_EVP_CMAC)
```

Comment:
Typo: `1` instead of `i` is compared against `D_EVP_CMAC`.

Reply:
Good catch!!

Reply:
My old eyes don't always fail me :smile: